src/test/java/ashe/TestFileToMinimize.java

```diff
@@ -0,0 +1,24 @@
+//package com.example;
```

Comment:
This file is an input, not a test source file. So, it should probably be stored under `src/test/resources` (not `src/test/java`); otherwise, the build system will quite reasonably start trying to compile it. I assume you've commented its contents out here because the build system issued some kind of complaint about this file if you did not.